packages/xod-cli/src/messages.js

```diff
@@ -1,9 +1,174 @@
+  `if two positions:\n` +
+  `\tfirst position is the path to a project/xodball\n` +
+  `\tsecond position is the patch name like 'main' or '@/main'`;
 
-// TODO: All messages for Users should be moved here
+export const openNotAXodFile = path => `Tried to open not a xod file: ${path}`;
+
+export const onBehalfFlagDescription = 'publish on behalf of the username';
+
+export const passwordFlagDescription = 'XOD API password';
+
+export const patchNotFound = 'Patch not found';
+
+export const patchNotValidPath = path => `Not a valid patch path: ${path}`;
+
+export const publishCommandAccessDenied = (username, libPath) =>
+  `user "${username}" can't access ${libPath}`;
+
+export const publishCommandAccessTokenNotFound =
+  'Access Token not found in reply';
+
+export const publishCommandAuthentication = 'Authenticate';
+
+export const publishCommandAuthError = msg =>
+  `Can't authenticate because of ${msg}`;
+
+export const publishCommandCreateLibrary = 'Create library';
+
+export const publishCommandDescription = 'publish a library';
+
+export const publishCommandExamples = [
+  'Publish the current project with the version defined in `project.xod`\n' +
+    '$ xodc publish',
+  'Publish a project saved as xodball\n$ xodc publish foo.xodball',
+];
+
+export const publishCommandGetLib = 'Get library';
+
+export const publishCommandGetUser = 'Get user';
+
+export const publishCommandLibGetError = (libname, msg) =>
+  `Can't get library ${libname} because of ${msg}`;
+
+export const publishCommandLibLoading = 'Load library';
+
+export const publishCommandLibPost = 'Publish library';
+
+export const publishCommandLibPostError = msg =>
+  `Can't publish library because of ${msg}`;
+
+export const publishCommandLibPutError = msg =>
+  `Can't create library because of ${msg}`;
+
+export const publishCommandUsage = 'publish [options] [entrypoint]';
+
+export const publishCommandUser404 = username =>
+  `User ${username} doesn't exist`;
+
+export const publishCommandUserError = msg =>
+  `Can't get user because of ${msg}`;
+
+export const publishCommandVersionExists = libpath =>
+  `Version ${libpath} already exists`;
+
+export const quietFlagDescription = 'do not log messages other than errors';
+
+export const resaveCommandDescription =
+  'opens a project and saves it in another location or format';
+
+export const resaveCommandExamples = [
+  `Exports the current multifile project to a xodball\n` +
+    `$ xodc resave . -o ~/foo.xodball\n`,
+  `Outputs the current multifile project as a xodball to stdout\n` +
+    `$ xodc resave\n`,
+  `Resaves one xodball into another (useful for applying migrations)\n` +
+    `$ xodc resave foo.xodball -o bar.xodball\n`,
+  `Converts a xodball to a multifile project\n` +
+    `$ xodc resave foo.xodball -o /some/new/dir\n`,
+];
+
+export const resaveCommandProjectLoading = 'Project loading';
+
+export const resaveCommandSaving = 'Saving';
+
+export const resaveCommandUsage = 'resave [options] [entrypoint]';
```

Comment:
I don’t think keeping `*Usage` in `messages.js` is a good idea. The file is for plain English phrases.